### PR TITLE
Initialized to use tito and added spec file

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
scratch builds here:
http://koji.katello.org/koji/taskinfo?taskID=216889
http://koji.katello.org/koji/taskinfo?taskID=216888

**Comment by jlsherrill** [comment]
Also tested on a 2.1 and install with this and it seemed to install with no errors.

File: foreman-selinux.spec [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+BuildRequires:  checkpolicy, selinux-policy-devel, hardlink
+BuildRequires:  policycoreutils >= %{selinux_policycoreutils_ver}
+BuildRequires:  /usr/bin/pod2man
+BuildArch:      noarch
+
+Requires:           selinux-policy >= %{selinux_policy_ver}
+Requires(post):     /usr/sbin/semodule, /sbin/restorecon, /usr/sbin/setsebool, /usr/sbin/selinuxenabled, /usr/sbin/semanage
+Requires(post):     policycoreutils-python
+Requires(post):     selinux-policy-targeted
+Requires(postun):   /usr/sbin/semodule, /sbin/restorecon
```
Comment (lzap) [inline foreman-selinux.spec]: We actually need Require: foreman-selinux with version specifier (current version + 1 - the removal was not yet merged).

If we have no time to wait, then without version for now.

**Comment by lzap** [comment]
LGTM, add the require please

File: foreman-selinux.spec (hunk 1 above)
Comment (jlsherrill) [inline foreman-selinux.spec]: Doesn't this work just fine with the current foreman-selinux ?     I'm not sure i get why the version specifier for 1.8 (i presume)

Comment (lzap) [inline foreman-selinux.spec]: We need it for elasticsearch port definition which will prevent *foreman* policy from loading (katello is not the problematic one in this case). See the referenced PR. We will likely bump the release number to 1.8.0-1 to get there.

**Comment by lzap** [comment]
Let's wait after FOSDEM, we need to bump version of the foreman package to get it working. Katello package can be only installed AFTER foreman-selinux was properly upgraded (and the port was undefined).

**Comment by jlsherrill** [comment]
@lzap good with merging?

**Comment by lzap** [comment]
I haven't tested yet.

**Pull request merged**